src/constants/courses.ts

```diff
@@ -2762,7 +2762,6 @@ export const courses: { [course: string]: string } = {
   BCSE304L: 'Theory of Computation',
   BCSE204L: 'Design and Analysis of Algorithms',
   BCSE204P: 'Design and Analysis of Algorithms Lab',
-  BECE309L: 'Artificial Intelligence and Machine Learning',
```

Comment:
Why did you remove this course?

Reply:
Multiple entry with the same id and value